pool_bin

```diff
@@ -582,6 +589,9 @@ def main():
     if args.cmd:
         func = args.func
         args = vars(args)
+        # only pass debug for pool-get
+        if func.__name__ != 'get' and "debug" in args:
+            del args["debug"]
```

Comment:
Should probably warn that unused args are unused imo

Reply:
Yeah, that's fair. What would you suggest? Just a `print()`? Or should I use the logger?

Reply:
Just had a closer look (after our offline chat about this) and it appears that the `--debug` switch only applies to `pool-get` already. If you try to use `--debug` with other commands, you get an error. E.g.:
```
pool-info --debug
usage: pool [-h] {exists,gc,get,info-get,info,init,list,register,unregister} ...
pool: error: unrecognized arguments: --debug
```